### PR TITLE
First draft of the new CoC

### PR DESCRIPTION
**Pull request by bryan-m-hughes**
This is a first, rough draft of the CoC. It is not yet complete, and so is not ready for merging, but I would like everyone to start getting your eyes on it. I ended up basing it primarily on the [Citizen Code of Conduct](http://citizencodeofconduct.org/), which the nodebots.io forum's CoC is also based on.

The most notable thing missing is contact information for people other than myself. I'm going to reach out to some other people, based on @dshaw's suggestions, in the morning to see if they are willing to act as points of contact. If you want to be a point of contact as well, *please let me know*!

Overall I'm pleased with direction the document is taking, but it currently doesn't do a good job of explaining the kind of community we want to have and, as a result, what our values are. I'm not sure that this CoC is the best place for that given its tone, so I'm starting to think about a separate document that is more positive, less specific, and less legalistic. What does everyone else think?

### CODE REVIEW

File: coc.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Contact Info
+
+If you need to report an incident, you can email us at incident@nodebotssf.org, call or text us at (415) 944-4941, or talk to any of the organizers at an event. In addition, you can contact any of the following people directly:
+
+Bryan Hughes, Co-organizer of NodeBots SF
+bryan@theoreticalideations.com
+https://twitter.com/nebrius
+
+[INSERT OTHERS HERE]
```
Comment (Frijol) [inline coc.md]: Add me! Kelsey Breseman, ifoundthemeaningoflife@gmail.com, https://twitter.com/selkeymoonbeam

Comment (bryan-m-hughes) [inline coc.md]: Love the email address!

File: coc.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+If you have been banned and show up at an event, you will be escorted off the premises by community organizers. Local law enforcement may be contacted, at the organizers' discretion.
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify us with a concise description of your grievance at the email address listed under Contact Info. Your grievance will be handled in accordance with our governing policies.
+
+## Contact Info
+
+If you need to report an incident, please call or text us at (415) 944-4941, email us at incident@nodebotssf.org, or talk to any of the organizers at an event. In addition, you can contact any of the following co-organizers directly:
+
+Emily Rose
+[Insert contact info here]
```
Comment (bryan-m-hughes) [inline coc.md]: @nexxy, can I get a public email address for you? Also, are you still on twitter, I couldn't seem to find you.

**Comment by bryan-m-hughes** [comment]
Hey everyone, after our discussions last Tuesday, I think we're ready to land this. This doesn't mean the CoC is "finished" or "locked", but usable enough that we can start pointing people to it. Please feel free to open issues/PRs/etc to update anything you think needs tweaking.

**Comment by dshaw** [comment]
:+1:

**Pull request merged**